src/Components/OATPropertyEditor/FormUpdateProperty.tsx

```diff
@@ -653,6 +637,26 @@ export const FormUpdateProperty = ({
                 />
             </div>
 
+            <div className={propertyInspectorStyles.modalRow}>
+                <Text styles={columnLeftTextStyles}>
+                    {t('OATPropertyEditor.id')}
+                </Text>
+                <TextField
+                    placeholder={t('OATPropertyEditor.id')}
+                    onChange={(_ev, value) =>
+                        handleIdChange(
```

Comment:
I think we're using the `onXXX` pattern for event handlers now. Can you please update this one?

Reply:
Also this is replicated to all the Utils functions we have in this class

Reply:
Good point. Want to open a task for us to review the naming in the Utils file at a future date?

Reply:
I think it would be better to do so